pkg/docker/progress.go

```diff
@@ -81,6 +81,7 @@ var (
 )
 
 // Special init function for this package
+// nolint
```

Comment:
@jbrekelmans could you please provide the justification

Reply:
Sure: 

> Use of init here is safe because the function has no side effects other than initializing some global variables (but since init runs first nothing can observe this). Since the values of the globals are inter-dependent, using code to initialize them encodes such dependencies and makes the code easier to understand for developers.